src/controller/CHIPDeviceControllerFactory.cpp

```diff
@@ -110,6 +110,10 @@ CHIP_ERROR DeviceControllerFactory::ReinitSystemStateIfNecessary()
 
 CHIP_ERROR DeviceControllerFactory::InitSystemState(FactoryInitParams params)
 {
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+    Transport::TcpListenParameters tcpListenParams;
```

Comment:
Why isn't this declared where it's assigned to?

Reply:
Yes, moved it. For some reason, I mistook the ReturnOnError() for a jump to the exit label.